### PR TITLE
Bao/enable locker

### PR DESCRIPTION
**Pull request by StarryFolf**
implemented enable locker, made some tests and refined some other stuffs

### CODE REVIEW

**Comment by StarryFolf** [comment]
fuck me, it happened again

File: src/Api/Middlewares/ExceptionMiddleware.cs [hunk 1]
```diff
@@ -82,6 +84,18 @@ private async void HandleRequestValidationException(HttpContext context, Excepti
         await context.Response.Body.WriteAsync(SerializeToUtf8BytesWeb(result));
     }
 
+    private async void HandleEntityNotAvailableException(HttpContext context, Exception ex)
+    {
+        context.Response.StatusCode = StatusCodes.Status404NotFound;
```
Comment (kaitoz11) [inline src/Api/Middlewares/ExceptionMiddleware.cs:89]: Should be something other than `404 not found`
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404

File: src/Application/Lockers/Commands/EnableLocker/EnableLockerCommand.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    {
+        var locker = await _context.Lockers
+            .Include(x => x.Room)
+            .FirstOrDefaultAsync(x => x.Id.Equals(request.LockerId), cancellationToken);
+        if (locker is null)
+        {
+            throw new KeyNotFoundException("Locker does not exist.");
+        }
+
+        if (locker.IsAvailable == true)
```
Comment (kaitoz11) [inline src/Application/Lockers/Commands/EnableLocker/EnableLockerCommand.cs:37]: `if(locker.IsAvailable)` is more than enough

**Review by kaitoz11 — COMMENTED** [review]
need @ChienNQuang or @Vzart to review and confirm...

File: src/Api/Middlewares/ExceptionMiddleware.cs (hunk 1 above)
Comment (StarryFolf) [reply src/Api/Middlewares/ExceptionMiddleware.cs:89]: chiến said use 404

Comment (kaitoz11) [reply src/Api/Middlewares/ExceptionMiddleware.cs:89]: use Confilct some thing with 409 status code and remove this shit

**Review by Vzart — APPROVED** [review]

**Pull request merged**